src/main/java/org/jitsi/videobridge/octo/OctoTentacle.java

```diff
@@ -311,31 +398,116 @@ public void expire()
      */
     public void sendMessage(String message)
     {
-        octoTransport.sendString(
+        bridgeOctoTransport.sendString(
             message,
             targets,
             conference.getGid()
         );
     }
 
+    public void addHandler(String epId, IncomingOctoEpPacketHandler handler)
+    {
+        logger.info("Adding handler for ep ID " + epId);
```

Comment:
I was thinking this isn't too bad, since it's just one per endpoint and, if for some reason we thought packets weren't flowing the first questions we would ask would be if the handler was put in so it seemed valuable.